model/post.go

```diff
@@ -10,11 +10,12 @@ import (
 )
 
 const (
-	POST_SYSTEM_MESSAGE_PREFIX = "system_"
-	POST_DEFAULT               = ""
-	POST_SLACK_ATTACHMENT      = "slack_attachment"
-	POST_JOIN_LEAVE            = "system_join_leave"
-	POST_HEADER_CHANGE         = "system_header_change"
+	POST_SYSTEM_MESSAGE_PREFIX  = "system_"
+	POST_DEFAULT                = ""
+	POST_SLACK_ATTACHMENT       = "slack_attachment"
+	POST_JOIN_LEAVE             = "system_join_leave"
+	POST_HEADER_CHANGE          = "system_header_change"
+	POST_OUT_OF_CHANNEL_MENTION = "system_out_of_channel_mention"
```

Comment:
Since this is an ephemeral message does it need this?  it isn't stored in the db.  Why not generic POST_EPHERMAL